.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -280,7 +280,7 @@ data class TestCoverage(
 
 enum class TestType(val unitTests: Boolean = true, val functionalTests: Boolean = true, val crossVersionTests: Boolean = false, val timeout: Int = 180, val maxParallelForks: Int = 4) {
     // Include cross version tests, these take care of selecting a very small set of versions to cover when run as part of this stage, including the current version
-    quick(true, true, true, 120, 2),
+    quick(true, true, true, 120, 8),
```

Comment:
❓ I suppose that changes the Linux tests as well. Do you have measurements there, too?